ChronosTankDrive/src/main/java/frc/robot/Robot.java

```diff
@@ -352,25 +352,22 @@ public void autonomousInit() {
         break;
       case "Shoot2":
         intake.getDeploy().set(true);
-        intake.getMotor().set(VictorSPXControlMode.PercentOutput, 1);
+        intake.getMotor().set(VictorSPXControlMode.PercentOutput, .6);
         shooter.getHood().resetHomed();
         start = System.currentTimeMillis();
         while (start + 3000 > System.currentTimeMillis()) {
           chronosDrive.tankDrive(-.7, -.7);
           shooter.getHood().homePeriodic();
         }
-        intake.getMotor().set(VictorSPXControlMode.PercentOutput, 1);
+        intake.getMotor().set(VictorSPXControlMode.PercentOutput, 0);
         intake.getDeploy().set(false);
-        delay(1500);
-        intake.getDeploy().set(true);
         start = System.currentTimeMillis();
-        while (start + 2500 > System.currentTimeMillis()) {
+        while (start + 500 > System.currentTimeMillis()) {
```

Comment:
This type of looping might spike the RoboRIO CPU. Try adding a 10 or 20 ms (or even longer) sleep to the loop and see if that helps.

Reply:
Loops like this occur several times, this should be a method, like `driveForTime(left, right, time);` or something similar.  I realize that you home the hood in some of these loops, and we should see if there's a better way of doing that, maybe by putting it in it's own thread.